### PR TITLE
Docs: Update docker installation statement.

### PR DESCRIPTION
**Pull request by xiaoxq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,15 +31,11 @@ More instructions are below
 
 ### The docker environment can be set by the commands below.
 
-Running a docker container from a linux system is done with this sequence. At the end, the `dev_into.sh`
-script will get you into the container
+First, you need to (install docker-ce properly)[https://github.com/ApolloAuto/apollo/blob/master/docker/scripts/README.md#install-docker].
```
Comment (zhxt) [inline README.md]: the Markdown syntax should be``[]()``
:-)

File: README_cn.md [hunk 2]
```diff
@@ -22,9 +22,10 @@ not because they are easy, but because they are hard.
 
 ### 设置Docker环境请运行如下命令
 
+首先，你需要(正确地安装Docker-ce)[https://github.com/ApolloAuto/apollo/blob/master/docker/scripts/README.md#install-docker].
```
Comment (zhxt) [inline README_cn.md]: ditto.

File: README.md (hunk 1 above)
Comment (xiaoxq) [reply README.md]: Oops. Nice catch!

File: README_cn.md (hunk 2 above)
Comment (xiaoxq) [reply README_cn.md]: Done.

**Pull request merged**